wagtailcaptcha/models.py

```diff
@@ -12,9 +12,7 @@
 class WagtailCaptchaEmailForm(AbstractEmailForm):
     """Pages implementing a captcha form with email notification should inhert from this"""
 
-    def __init__(self, *args, **kwargs):
-        super(WagtailCaptchaEmailForm, self).__init__(*args, **kwargs)
-        self.form_builder = WagtailCaptchaFormBuilder
+    form_builder = WagtailCaptchaFormBuilder
```

Comment:
I tried creating a base mixin that both class could use, but I ran into issues (migrations can't serialize the class or manager isn't available because the class is abstract) when trying to inherit both from the mixin and Wagtail abstract classes so I kept it simple instead. There isn't much code anyway.